vmdb/app/models/mixins/ems_openstack_mixin.rb

```diff
@@ -24,7 +24,7 @@ def browser_url
   def openstack_handle(options = {})
     require 'openstack/openstack_handle'
     @openstack_handle ||= begin
-      raise "no credentials defined" if self.missing_credentials?(options[:auth_type])
+      raise MiqException::MiqInvalidCredentialsError, "No credentials defined" if self.missing_credentials?(options[:auth_type])
```

Comment:
I don't know, I didn't get to normalizing the exceptions yet.  

But, I like this change, because it seems to fall in line with what authetication_mixin knows how to handle (MiqException::MiqInvalidCredentialsError raised by verify_credentials)
https://github.com/jrafanie/manageiq/blob/ce3bf060f87757689a606d247f332ee40b62ffc3/vmdb/spec/models/mixins/authentication_mixin_spec.rb#L509